tests/schemas/api_v3/tokenTopBalances.schema.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+import Joi from 'joi';
+
+const tokenTopBalancesMetaSchema = {
+	count: Joi.number().integer().min(0).required(),
+	offset: Joi.number().integer().min(0).required(),
+	total: Joi.number().integer().min(0).required(),
+};
+
+const goodResponseSchemaForTokenTopBalances = Joi.object({
+	data: Joi.object().required(),
```

Comment:
We can also define data schema for the given endpoint. We can discuss offline if needed.

Reply:
Applied